rts/Game/Camera/CameraController.cpp

```diff
@@ -25,9 +24,26 @@ CCameraController::CCameraController()
 
 	pos = float3(mapDims.mapx * 0.5f * SQUARE_SIZE, 1000.0f, mapDims.mapy * 0.5f * SQUARE_SIZE); // center map
 	dir = FwdVector;
+
+	configHandler->NotifyOnChange(this, {"CameraMoveFastMult"});
```

Comment:
Can we move listeners to `CameraMoveFastMult` and `CameraMoveSlowMult` inside `CCamera` instead?

Then for each controller that requires access to them we can simply `camera->moveSlowMult` and `camera->moveFastMult`

Reply:
Interesting, this sounds like a reasonable compromise to me. :+1:

Reply:
Especially since camerahandler seems to be the only place where camera is initialized.